src/scripts/lexmatch-sssom-compare.py

```diff
@@ -303,7 +303,6 @@ def get_unmapped_df(
         new_df['confidence'] = new_df['confidence'].apply(lambda x: x if 0 <= x <= 1 else 0.5)  # Replace out of range values with default
         new_df['confidence'].fillna(0.5, inplace=True)  # Replace NaN with default value
 
-    new_df.to_csv("check_me_out.sssom.tsv", sep="\t")
```

Comment:
Thought we got rid of this

Reply:
Yes, it was removed from both `main` and `develop` https://github.com/monarch-initiative/mondo-ingest/commit/259725ba558eca8bbce4f392b4b01b98dae3bb74

Reply:
I was probably not synced. Now I am!